onnxruntime/python/tools/kernel_explorer/kernels/batched_gemm_test.py

```diff
@@ -0,0 +1,210 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import sys
+from dataclasses import dataclass
+from itertools import product
+
+import kernel_explorer as ke
+import numpy as np
+import pytest
+from utils import get_gemm_basic_sizes, get_gemm_bert_sizes, get_gemm_bound, transab_to_suffix
+
+
+def dtype_to_suffix(dtype):
+    return {
+        "float32": "float",
+        "float16": "half",
+    }[dtype]
+
+
+def _test_batched_gemm(
+    func, dtype: str, transa: bool, transb: bool, m: int, n: int, k: int, batch: int, alpha=1.0, beta=0.0
+):
+    assert dtype in ["float32", "float16"]
+
+    a_shape = (k, m) if transa else (m, k)
+    b_shape = (n, k) if transb else (k, n)
+
+    np.random.seed(0)
+    as_ = [(np.random.rand(*a_shape) + 0.5).astype(dtype).astype("float64") for i in range(batch)]
+    bs = [(np.random.rand(*b_shape) + 0.5).astype(dtype).astype("float64") for i in range(batch)]
```

Comment:
why they named as `as_` and `bs` instead of `as_` and `bs_`?

Reply:
`as` is a keyword in python...